internal/app/machined/pkg/system/services/dashboard.go

```diff
@@ -66,7 +66,7 @@ func (d *Dashboard) Runner(r runtime.Runtime) (runner.Runner, error) {
 		}),
 		runner.WithStdinFile(tty),
 		runner.WithStdoutFile(tty),
-		runner.WithCtty(1),
+		runner.WithCtty(0),
```

Comment:
ctty(1) failed as stdout is opened as write-only. We might want to change that, but perhaps using stdin as ctty is perfectly fine as we pass both for TTY to work